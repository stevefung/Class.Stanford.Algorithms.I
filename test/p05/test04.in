1 2,8 3,17
2 1,7 3,10 4,5
3 1,12
4 3,3
