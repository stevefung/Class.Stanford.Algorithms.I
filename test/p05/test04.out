Vert #1, dist 0 from -1
Vert #2, dist 8 from 1
Vert #3, dist 16 from 4
Vert #4, dist 13 from 2
